pom.xml

```diff
@@ -138,6 +139,8 @@
     <!-- timestamp for reproducible outputs, updated on release by the release plugin -->
     <project.build.outputTimestamp>2024-07-29T06:03:16Z</project.build.outputTimestamp>
     <rat.consoleOutput>true</rat.consoleOutput>
+    <revision>2.1.4</revision>
+    <sha1 />
```

Comment:
```suggestion
    <sha1/>
```

Reply:
This change would violate our formatter rules, which were written that way to be consistent with the way several different plugins interact with the POM. The space would be correct.

Reply:
> This change would violate our formatter rules

I just tired making this change locally and running the build added the space back.

Reply:
ah yep, I expanded the file and see everything else following the single space format. 
carry on 👍